llvm/test/tools/sycl-post-link/emit_imported_symbols.ll

```diff
@@ -0,0 +1,94 @@
+; This test checks that the -emit-imported-symbols option generates a list of imported symbols
+; Function names were chosen so that no function with a 'inside' in their function name is imported
+;
+
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+; Test with -split=kernel
+;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
+
+; RUN: rm -f %t*.prop %t*.sym
```

Comment:
Any temporary files created by LIT tests are automatically cleaned up by LIT, this line is not needed

Reply:
Some temporary files persist across llvm-lit invocations.  RM line removed to be consistent with our test conventions.  Will separately investigate if there is a way to invoke llvm-lit to fully remove temp files.